public/app/features/inspector/InspectStatsTab.tsx

```diff
@@ -73,3 +74,12 @@ export const InspectStatsTab = ({ data, timeZone }: InspectStatsTabProps) => {
     </div>
   );
 };
+
+const getStyles = () => {
+  return {
+    container: css`
+      height: 100%;
+      overflow-y: scroll;
+    `,
+  };
+};
```

Comment:
nit: this can technically just be inlined as is not depending on anything: 
```suggestion
const containerStyle = css`
  height: 100%;
  overflow-y: scroll;
`;
```

Reply:
Good idea!